app/router.js

```diff
@@ -3,7 +3,19 @@ import config from './config/environment';
 
 const Router = Ember.Router.extend({
     location: config.locationType,
-    rootURL: config.rootURL
+    rootURL: config.rootURL,
+    willTransition(oldInfos, newInfos) {
+        const disableList = ['settings', 'moderation'];
+        const prevRoute = oldInfos.length ? oldInfos[oldInfos.length - 1].name.split('.') : null;
+        const prevName = prevRoute != null ? prevRoute[prevRoute.length - 1] : null;
+        const currentRoute = newInfos.length ? newInfos[newInfos.length - 1].name.split('.') : null;
+        const currentName = currentRoute != null ? currentRoute[currentRoute.length - 1] : null;
+        const disableResetScroll = disableList.includes(prevName) && disableList.includes(currentName);
+        if (!disableResetScroll) {
+            this._super(...arguments);
```

Comment:
I think it should always call `_super`, even if it doesn't scroll to the top. Unless there's something I'm missing?